.github/workflows/maven-lib-release.yml

```diff
@@ -1,157 +1,10 @@
-        run: | 
-          git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-          git fetch --all
-          git merge origin/release
-          git push origin ${GITHUB_REF/refs\/heads\//}
-          artifactId=$(xmlstarlet pyx pom.xml | grep -v ^A | xmlstarlet p2x | xmlstarlet sel -t -v "/project/artifactId")
-          version=$(xmlstarlet pyx pom.xml | grep -v ^A | xmlstarlet p2x | xmlstarlet sel -t -v "/project/version")
-          artifact="${artifactId}-${version}.jar"
-          git tag "v$version"
-          git push --tags
-          echo "::set-output name=tag::$(echo v$version)"
-          echo "::set-output name=artifact::$(echo $artifact)"
-          echo "::set-output name=artifact-path::$(echo target/$artifact)"
-        shell: bash
-        continue-on-error: true
-
-      - name: Fail
-        if: steps.merge.outcome == 'failure'
-        run: | 
-          git push origin -d release
-          echo "There was an error in merging the branch. release branch was deleted."
-          exit 1
-        shell: bash
-
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ steps.merge.outputs.tag }}
-          release_name: ${{ steps.merge.outputs.tag }}
-          body_path: changelog.md
-          draft: false
-          prerelease: false
-        continue-on-error: true
-
-      - name: Download binary
-        uses: actions/download-artifact@v2
-        with:
-          name: ${{ needs.deploy-central.outputs.artifact }}
-          path: built-artifacts
-        continue-on-error: true
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: built-artifacts/${{ needs.deploy-central.outputs.artifact }}
-          asset_name: ${{ needs.deploy-central.outputs.artifact }}
-          asset_content_type: application/zip
-        continue-on-error: true
-
-  update-version:
-    needs: create-release
-    runs-on: ubuntu-latest
-    container: zepben/pipeline-basic:5.5.0
-    env:
-      NEXUS_MAVEN_REPO: ${{ secrets.NEXUS_MAVEN_REPO }}
-      NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
-      NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
-      NEXUS_MAVEN_SNAPSHOT: ${{ secrets.NEXUS_MAVEN_SNAPSHOT }}
-      NEXUS_MAVEN_RELEASE: ${{ secrets.NEXUS_MAVEN_RELEASE }}
-      SLACK_NOTIFICATION: YES
-      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Update to next minor version
-        run: |
-          /scripts/update-version.sh --java --maven --release pom.xml changelog.md
-        shell: bash
+  run:
+    uses: zepben/.github/.github/workflows/maven-lib-release.yml@main
+    with:
+      private: false
```

Comment:
We need to make maven-lib-release also support the licencecheck action.

You can use the python-lib-release.yml as a reference in the .github as I implemented it there already.

Once done that you'll need to add `licencecheck: true` here

Reply:
yup, no worries